wordpress/wp-content/plugins/memberful-wp/src/metabox.php

```
@@ -55,6 +55,10 @@ function memberful_wp_metabox( $post ) {
   $view_vars['viewable_by_any_registered_users'] = memberful_wp_get_post_available_to_any_registered_users( $post->ID );
   $view_vars['viewable_by_anybody_subscribed_to_a_plan'] = memberful_wp_get_post_available_to_anybody_subscribed_to_a_plan( $post->ID );
 
+  $paywall_config = Memberful_Paywall_Config::get();
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'Memberful_Paywall_Config' in method 'memberful_wp_metabox'. (undefined)

(StaticAccess)